docs/developer-docs/devquests.md

```diff
@@ -2,21 +2,21 @@
 
 ## What are DevQuests?
 
-They're achievement development-related quests that developers can take part in at any time. Each one has a specific goal and upon reaching that goal, the achievement developer will earn the badge.
+They're achievement development-related quests that Developers can take part in at any time. Each one has a specific goal and upon reaching that goal, the achievement developer will earn the badge. DevQuests are not available for Junior Developers.
 
 ## General Rules
 
-1. For Set-Creation achievements other than DQ15, please announce your DQ pick in devquest or via the DevQuest inbox on site early on in your development. This will give us time to make sure any issues with the choice can be sorted out early. Late requests may not count.
-2. For Maintenance achievements (i.e.: DQ1, DQ5, DQ14, etc), just create a column in the DevQuest Remastered sheet to the left of completed entries and fill it in as you work. Contact the DevQuest team when you are complete with all entries for review.
-3. For Revision achievements (i.e.: DQ2 and DQ13), make sure your choice is already on the corresponding approved list. You will need to satisfy the repair plan. These quests should be more than simple one-achievement revisions. If a repair plan seems to be overly simple, contact us first to ensure it will qualify.
+1. For Set-Creation achievements other than DevQuest 015 and DevQuest 020, announce your DQ pick in [`#devquest`](https://discord.com/channels/310192285306454017/310195377993416714) or via the DevQuest inbox on site early on in your development. This will give us time to make sure any issues with the choice can be sorted out early. Late requests may not count.
```

Comment:
Recommend changing "Late requests may not count" to "Late requests may not be counted".